### PR TITLE
DAT-60126 ecomm app run mode edit calendar events

### PR DESCRIPTION
**Pull request by etrubek**
Added topFiltersRenderer function enable the option to add top components.
Also, added className to body rows

### CODE REVIEW

File: app/src/components/base/Table.js [hunk 1]
```diff
@@ -57,7 +57,8 @@ const Table = props => {
     placeholder,
     footerText,
     emptyRenderer,
-    onSearch
+    onSearch,
+    topFiltersRenderer
```
Comment (Chikel) [inline app/src/components/base/Table.js:61]: I think we better call it something like `tableHeaderRenderer` as `topFiltersRenderer` is specific for your use case.
But - looking at Abstract, I don't think we should render the filters as part of the table, these can just be on top of the table and you can remove the Table's search input.
Another renderer you'll probably need is `columnHeaderRenderer` for adding the sort icons.

Comment (etrubek) [reply app/src/components/base/Table.js:61]: 1.Shay picked the name, topFiltersRenderer, I don't mind.
2. Regarding rendering the filters on top of the table, I suggested that option too, but we decided to go with topFiltersRenderer, again I don't mind.
3. I didn't use ColRenderer for the selectAll checkbox, don't have to use it here too.

Comment (Chikel) [reply app/src/components/base/Table.js:61]: @etrubek, ok then. I'm still not sure how you plan on adding the sorting icons to the column headers but I trust your judgment 👍

Comment (crazypixel) [reply app/src/components/base/Table.js:61]: @etrubek @Chikel 
let's change the name `tableHeaderRenderer` is better.
I agree with aran the filters can be outside the table component.
for the desc / asc icons. yes you need to add a `columnHeaderRenderer`

**Comment by etrubek** [comment]
@crazypixel, changed name and updated version,
Looks find without colRenerre:
image

**Pull request merged**